log.py

```diff
@@ -12,5 +13,8 @@ def printReport(self):
         for key in self.record_:
             print(f"Type: {key} Value: {self.record_[key]}")
 
+    def output(self, message):
```

Comment:
Why is message a function instead of a string?

Reply:
So that the strings are not evaluated if the logging is disabled

Reply:
Interesting; I wonder how creating a closure function compares to creating a string. It seems like it's more efficient but you never know with python